openassessment/assessment/api/peer.py

```diff
@@ -37,6 +37,23 @@ def flexible_peer_grading_enabled(peer_requirements, course_settings):
     return peer_requirements.get("enable_flexible_grading")
 
 
+def mean_peer_grading_enabled(peer_requirements):
+    """
+    Is mean peer grading turned on? Either at the course override
+    level or the block level?
+    """
+    return peer_requirements.get("enable_mean_grading")
+
+
+def get_peer_score_type(peer_requirements):
+    """
+    Get the peer grading type, either mean or median.
+    """
+    if mean_peer_grading_enabled(peer_requirements):
+        return "mean"
+    return "median"
```

Comment:
As we are opening the door for other grading strategies (and other folks have shown interest on different grading strategies), can you update the implementation from "enable_mean_grading" to a more general field "peer_grading_strategy" or "grading_strategy" using a enum that defines the which grading strategies are available.

Of course, as this is a POC. It's not necessary, till we receive more feedback from the ORA team